### ISSUE TITLE
Deprecation warnings on install

### ISSUE THREAD

**Issue by silverwind** [jedmao/eclint#114]
Installing `eclint` via `yarn` currently leaves these four deprecations warnings:

````
warning eclint > gulp-util@3.0.8: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
warning eclint > gulp-filter > gulp-util@3.0.8: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
warning eclint > editorconfig > @types/commander@2.12.2: This is a stub types definition for commander (https://github.com/tj/commander.js). commander provides its own type definitions, so you don't need @types/commander installed!
warning eclint > gulp-reporter > goo.gl > request > node-uuid@1.4.8: Use uuid module instead
````

**Comment by gucong3000** [comment]
Thank you for your report.
Fixed in baster branch, and to be release in 2018

**Comment by gucong3000** [comment]
Thank you for your report. I'll fix the problem next week.

**Comment by gucong3000** [comment]
fixed in #115
